### PR TITLE
qm-guest-cmd: add page

### PR DESCRIPTION
**Pull request by abhisheksrivastava177**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/qm-guest-cmd.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# qm guest cmd
+
+> Execute Qemu Guest Agent commands.
```
Comment (kbdharun) [inline pages/linux/qm-guest-cmd.md:3]:
```suggestion
> Execute QEMU Guest Agent commands.
```

File: pages/linux/qm-guest-cmd.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Execute Qemu Guest Agent commands:
```
Comment (kbdharun) [inline pages/linux/qm-guest-cmd.md:6]:
```suggestion
- Execute QEMU Guest Agent commands:
```

File: pages/linux/qm-guest-cmd.md (hunk 1 above)
Comment (abhisheksrivastava177) [reply pages/linux/qm-guest-cmd.md]: Done

File: pages/linux/qm-guest-cmd.md (hunk 2 above)
Comment (abhisheksrivastava177) [reply pages/linux/qm-guest-cmd.md]: Done

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/qm-guest-cmd.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# qm guest cmd
+
+> Execute QEMU Guest Agent commands.
+> More information: <https://pve.proxmox.com/pve-docs/qm.1.html>.
+
+- Execute QEMU Guest Agent commands:
```
Comment (navarroaxel) [inline pages/linux/qm-guest-cmd.md:6]:
```suggestion
- Execute a specific QEMU Guest Agent command:
```

Comment (abhisheksrivastava177) [reply pages/linux/qm-guest-cmd.md]: Done

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Comment by abhisheksrivastava177** [comment]
Thanks for merging my PR. This is my first OSS contribution. 😄

**Pull request merged**